featuretools/tests/primitive_tests/test_primitive_base.py

```diff
@@ -0,0 +1,9 @@
+from featuretools.primitives import Max
+
+
+def test_call():
```

Comment:
can you add a test where the primitive takes multiple arguments?

Reply:
add a test for both an aggregation and transform primitive

Reply:
add a test for primitives that have `uses_calc_time=True`  and get a `time` argument